osidb/models.py

```diff
@@ -1518,6 +1518,8 @@ def bzsync(self, *args, bz_api_key, **kwargs):
         # fetch from Bugzilla
         fc = FlawCollector()
         fc.sync_flaw(self.bz_id)
+        # Make sure the flaw instance has the latest data
+        self.refresh_from_db()
```

Comment:
I think that recently with @jobselko we dealt with something in Assembler collectors and this would have probably fixed it in a better way. I am not sure what exactly it was but something with difference between runtime and DB after the Bugzilla sync. So just FYI @jobselko

Reply:
Yeah, a "runtime" `flaw` with uuid `1` had different data than the same flaw retrieved by `Flaw.objects.get(uuid=1)` :).